templates/provider/hmc-templates/files/templates/azure-hosted-cp.yaml

```diff
@@ -2,6 +2,8 @@ apiVersion: hmc.mirantis.com/v1alpha1
 kind: ClusterTemplate
 metadata:
   name: azure-hosted-cp
+  labels:
```

Comment:
I don't think we want to label those in the system namespace as managed. Only distributed once.